packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -269,6 +279,12 @@ const NotificationDetail = () => {
     }
   }, [legalFactDownloadUrl]);
 
+  useEffect(() => {
```

Comment:
I would remove the useEffect and the setTimeoutMessage to avoid an unecessary refresh of the component